k8s/object.bzl

```diff
@@ -120,10 +120,14 @@ def _impl(ctx):
         output = ctx.outputs.executable,
     )
 
-    return struct(runfiles = ctx.runfiles(files = [
-        ctx.executable.resolver,
-        ctx.outputs.substituted,
-    ] + list(ctx.attr.resolver.default_runfiles.files) + all_inputs))
+    return [
+        DefaultInfo(
+            runfiles = ctx.runfiles(files = [
+                ctx.executable.resolver,
+                ctx.outputs.substituted,
+            ] + list(ctx.attr.resolver.default_runfiles.files) + all_inputs),
```

Comment:
ctx.attr.resolver.default_runfiles.files should be accessed in the new provider way as well. i.e. ctx.attr.resolver[DefaultInfo].default_runfiles.files

Reply:
Also use transitive_files attribute like this https://github.com/bazelbuild/rules_docker/pull/724/files#diff-0ebb096ed27e3e315a65e0f87e1a5bf9R53

Reply:
Done